content/docs/components/hyperparameter-tuning/hyperparameter.md

```diff
@@ -290,14 +290,24 @@ View the results of the experiment in the Katib UI:
 Run the following command to launch an experiment using the Kubeflow's 
```

Comment:
Change "command" to "commands", as there are now multiple commands.

Reply:
Done! do you want me to squash it?

Reply:
Thanks! No squashing needed.